state/state_test.go

```diff
@@ -322,7 +322,8 @@ func TestProposerFrequency(t *testing.T) {
 		vals := make([]*types.Validator, N)
 		totalVotePower := int64(0)
 		for j := 0; j < N; j++ {
-			votePower := int64(cmn.RandInt() % maxPower)
+			// make sure votePower > 0
+			votePower := int64(cmn.RandInt()%maxPower) + 1
```

Comment:
Can't this (rarely) exceed maxPower now?

Reply:
yes, and I think with +1 we can get 1000 but no more.

Reply:
You are right! Sorry for the noise.